socorro/external/postgresql/base.py

```diff
@@ -85,7 +85,7 @@ def query(self, sql, params=None, error_message=None, connection=None):
             if not connection:
                 connection = self.database.connection()
                 fresh_connection = True
-            # self.context.logger.debug(connection.cursor.mogrify(sql, params))
+            # logger.debug(connection.cursor().mogrify(sql, params))
```

Comment:
Historically we have always showed ``debug`` in our production logs, and that has been a pain. But we could easily change that I suppose. 

The other reason I can think about is that this is an option that you only want to turn on when you are debugging those SQL queries, which is not always the case. It's quite painful to have to go through a lot of log when debug something unrelated, so I think less log is better. 

But maybe it's just convenient to me because I know exactly where to go to turn this debugging on...